src/routes/postSkipSegments.js

```diff
@@ -232,9 +232,9 @@ module.exports = async function postSkipSegments(req, res) {
 
             try {
                 db.prepare("INSERT INTO sponsorTimes " + 
-                    "(videoID, startTime, endTime, votes, UUID, userID, timeSubmitted, views, category, shadowHidden)" +
-                    "VALUES(?, ?, ?, ?, ?, ?, ?, ?, ?, ?)").run(videoID, segmentInfo.segment[0], 
-                    segmentInfo.segment[1], startingVotes, UUID, userID, timeSubmitted, 0, segmentInfo.category, shadowBanned);
+                    "(videoID, startTime, endTime, votes, UUID, userID, timeSubmitted, views, category, shadowHidden, hashedVideoID)" +
+                    "VALUES(?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)").run(videoID, segmentInfo.segment[0],
+                    segmentInfo.segment[1], startingVotes, UUID, userID, timeSubmitted, 0, segmentInfo.category, shadowBanned, getHash(videoID, 0));
```

Comment:
`getHash(videoID, 0)` will result in no passes on the string for hashing (returning an empty string to protect data that should be hashed). Given the DB migration script, it should be 1.
https://github.com/ajayyy/SponsorBlockServer/blob/master/src/utils/getHash.js#L4

Recommend creating some tests that check the hashing of the video id is being done correctly on insert.

Likewise all the uses in the test scripts need to be changes.

Reply:
The test scripts should probably be hardcoded instead of being dependent on `getHash`